tests/func/test_api.py

```diff
@@ -126,3 +128,39 @@ def test_open_not_cached(dvc):
     os.remove(metric_file)
     with pytest.raises(FileMissingError):
         api.read(metric_file)
+
+
+def test_summon(tmp_dir, erepo_dir, dvc, monkeypatch):
+    artifacts_yaml = ruamel.yaml.dump(
+        {
+            "artifacts": [
+                {
+                    "name": "sum",
+                    "description": "Add <x> to <number>",
+                    "file": "calculator.py",
+                    "method": "add_to_num",
+                    "params": {"x": 1},
+                    "deps": ["number"],
```

Comment:
There needs to be `"summon": {"type": "python", "call": ...}` section. On any other type this should show appropriate error message.